include/seqan/basic/metaprogramming_math.h

```diff
@@ -142,7 +142,7 @@ template <uint64_t base> struct LogNFloor<0, base> { static const uint64_t VALUE
  * @tparam BASE     The base of the term (<tt>int64_t</tt>).
  * @tparam EXPONENT The exponent of the term (<tt>int64_t</tt>).
  *
- * @return uint64_t b<sup>e</sup
+ * @return uint64_t <sup>base</sup>
  *
```

Comment:
I think it should be: <tt>b<sup>e</sup></tt>, with b being the <tt>BASE</tt> and e the <tt>EXPONENT</tt>.